Doc/tutorial/stdlib.rst

```diff
@@ -72,10 +72,35 @@ three`` at the command line::
    >>> print(sys.argv)
    ['demo.py', 'one', 'two', 'three']
 
-The :mod:`getopt` module processes *sys.argv* using the conventions of the Unix
-:func:`getopt` function.  More powerful and flexible command line processing is
-provided by the :mod:`argparse` module.
+The :mod:`argparse` module provides a mechanism to process command line arguments.
+It should always be preferred over directly processing ``sys.argv`` manually.
 
+Take, for example, the below snippet of code::
+
+   >>> import argparse
+   >>> parser = argparse.ArgumentParser(description='An argparse example.')
+   >>> parser.add_argument('name', help='The name of someone to greet.')
+   >>> args = parser.parse_args()
+   >>> print(f'Hello, {args.name}!')
+
+The three lines of code needed for creating the command line argument formula
+are simple to read and understand and are easy to extend in terms of your
+applications functionality.
```

Comment:
I think you should just let the example stand, instead of asserting that it's simple to read and extend.

Reply:
I was attempting to make obvious my point, but if the examples can stand on their own and have the same effect then I'll gladly omit the statement :+1: